diesel/src/sqlite/types/date_and_time/chrono.rs

```diff
@@ -127,42 +128,86 @@ mod tests {
     fn unix_epoch_encodes_correctly() {
```

Comment:
Do we really need this test twice? The same value is also tested by the next test.

Reply:
This is testing serialization. The next test is testing deserialization.